base/libgit2/blob.jl

```diff
@@ -30,7 +30,7 @@ function lookup(repo::GitRepo, oid::GitHash)
     @check ccall((:git_blob_lookup, :libgit2), Cint,
```

Comment:
Actually, this function is redundant: we should deprecate it in favour of `GitBlob(repo::GitRepo, oid::GitHash)`.